qt/applications/workbench/workbench/plotting/test/test_plothelppages.py

```diff
@@ -0,0 +1,82 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench.
+import unittest
+from unittest.mock import ANY, patch
+from matplotlib import use as mpl_use
+mpl_use('Agg')  # noqa
+import numpy as np
+from plotting.plothelppages import *
+from matplotlib.pyplot import figure
+
+
+class PlotHelpPagesTest(unittest.TestCase):
```

Comment:
Good job adding tests for this! The file needs to be added to the `CMakeLists.txt` so that it is found by ctest. I think the one to add it to is in `mantid\qt\applications\workbench`.

Reply:
Ah yes, completely slipped my mind  🤦‍♂️ .